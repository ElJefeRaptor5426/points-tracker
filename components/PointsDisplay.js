import React from 'react';
import { StyleSheet, View, Text } from 'react-native';
import AppContext from '../AppContext';
import Points from './Points';

export default class PointsDisplay extends React.Component {
  render() {
    return (
      <AppContext.Consumer>
        {context => (
          <View style={styles.display}>
            <Text>Points:</Text>
            <Points points={context.points} />
          </View>
        )}
      </AppContext.Consumer>
    );
  }
}

const styles = StyleSheet.create({
  display: {
    color: '#000',
    fontSize: 42,
    textAlign: 'center',
  }
})