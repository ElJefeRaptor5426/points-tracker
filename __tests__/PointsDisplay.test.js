import React from 'react';
import { shallow } from 'enzyme';
import { Text } from 'react-native'
import PointsDisplay from '../components/PointsDisplay';

describe('PointsDisplay component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<PointsDisplay />, {points: 0})
    expect(wrapper).toMatchSnapshot();
  });
})