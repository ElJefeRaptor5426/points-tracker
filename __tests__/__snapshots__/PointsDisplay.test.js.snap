// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PointsDisplay component should render correctly 1`] = `ShallowWrapper {}`;
